app/assets/javascripts/application.js

```diff
@@ -75,5 +75,15 @@ angular.module("OpenHq", ['ngRoute', 'ngAnimate', 'ngSanitize', 'rails', 'ngFile
 
   // Don’t convert attributes to camel case
   railsSerializerProvider.underscore(angular.identity).camelize(angular.identity);
+})
 
+.run(function($rootScope, $mdDialog) {
```

Comment:
Maybe wrap it up in a service called `confirmDialog`? Seems a bit wrong attaching it to the root scope